src/python/PSetTweaks/PSetTweak.py

```diff
@@ -394,9 +400,9 @@ def unpersist(self, filename, formatting=None):
 
 
             jsoniser = JSONiser()
-            jsoniser.dejson(json.load(StringIO.StringIO(jsonContent)))
+            jsoniser.dejson(json.load(io.StringIO(jsonContent.decode("utf-8"))))
```

Comment:
I dealt with this line differently in:
https://github.com/dmwm/WMCore/pull/8733/files#diff-b600d610cf56591819a4df56f024cb45R397

and honestly, I'm not sure whether we should deal with that as a byte streaming, or decoding a text object as a unicode string.

Reply:
Uh, now I see. You are totally right on this, my solution is a waste of resources. 
What should I do? Add a commit that is the same as yours so that the PR, in case they will be both merged, will not collide? Should I open the PR without any change to this line?

Reply:
I'm actually not sure I'm right ;)
You deal with this following the sandwich approach, which sooner or later will need to be implemented (first, we need to define which encoding we use in the repo such that we do not need to start guessing how to encode/decode things).

Reply:
For now, I'd suggest you to rebase your PR and simply remove this fix.

Reply:
In this particular case, we are lucky: `json.loads`, which deserializes a JSON document into a python dictionary parsing, can have as input either a bytes string or a unicode string in both py2 and py3.

This means that we can just use
```
jsoniser.dejson(json.loads(jsonContent))
```

References
* https://docs.python.org/2.7/library/json.html#json.loads
* https://docs.python.org/3/library/json.html#json.loads

If we were not so lucky (if we had to deal with a library not as smart as `json`), we could have followed the suggestions from [future cheatsheet about py2 and py3 compatible file opening syntaxes](http://python-future.org/compatible_idioms.html#file-io-with-open), so that we could have been sure that `jsonContent` was a handle to bytes strings and that it could have been a proper input for `io.BytesIO()` in both py2 and py3.

Reply:
Perfect!